activemq-unit-tests/src/test/java/org/apache/activemq/security/AbstractCachedLDAPAuthorizationMapLegacyTest.java

```diff
@@ -322,7 +322,7 @@ public boolean isSatisified() throws Exception {
                     return map.context == null;
                 }
             }
-        });
+        }, 30*60*1000);
```

Comment:
10 minutes seems like an unreasonable amount of time to wait for a single test. The CI build is super long already. Do you think it could be shorter?